src/Axe.Windows.Fakes.Prebuild/Fakes.Prebuild.csproj

```diff
@@ -7,8 +7,8 @@
     <ProjectGuid>{149576B1-9947-4BE0-9008-DFFD0DA36E00}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>AccessibilityInsights.Fakes.Prebuild</RootNamespace>
```

Comment:
Build fails if FAKES_SUPPORTED=1 is set in the environment. Not really sure why...

Reply:
cause the fakes files weren't renamed too.